docs/how-to-setup-freecodecamp-locally.md

```diff
@@ -1,5 +1,7 @@
 # Set up freeCodeCamp locally
 
+As of 8 March 2018, please consider helping us test our new guide on how to [setup freeCodeCamp locally using Docker](/docs/how-to-setup-freecodecamp-locally-using-docker.md) instead of using this guide. It *should* result in fewer or no error but we won't know until enough devs try it.
```

Comment:
```suggestion
As of 8 March 2019, please consider helping us test our new guide on how to [setup freeCodeCamp locally using Docker](/docs/how-to-setup-freecodecamp-locally-using-docker.md) instead of using this guide. It *should* result in fewer or no errors. Thanks a lot for helping us make everyone's contributing experience better.
```

Reply:
Is it 2019 already? :-D